.github/workflows/ci.yml

```diff
@@ -29,6 +29,8 @@ jobs:
         include:
           - os: ubuntu-latest
             target: x86_64-unknown-linux-gnu
+          - os: ubuntu-latest
```

Comment:
Lets not create a new job run here - it requires a lot of CI resources. Instead, just add an additional deb build step or if-condition as part of the `ubuntu-latest / x86_64-unknown-linux-gnu`